src/Controls/src/Core/PerformanceTracker/PerformanceFeature.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.Maui.Controls.PerformanceTracker;
+
+/// <summary>
+/// Controls whether performance monitoring features are enabled.
+/// This class is used by the trimmer to remove performance monitoring code in production builds.
+/// </summary>
+internal class PerformanceFeature
+{
+	/// <summary>
+	/// Indicates whether performance monitoring is enabled.
+	/// </summary>
+	internal static bool IsSupported { get; }
```

Comment:
Cannot use `[FeatureSwitchDefinition("System.Diagnostics.Metrics.Meter.IsSupported")]` here because FeatureSwitchDefinition is internal.

Reply:
from @jonathanpeppers 

I think you just need 

#if NET9_0_OR_GREATER
[FeatureSwitchDefinition("Microsoft.Maui.RuntimeFeature.IsIVisualAssemblyScanningEnabled")]